Documentation/teaching/info/contributing.rst

```diff
@@ -70,7 +70,7 @@ Forking the repository
    .. code-block:: bash
 
      $ mkdir -p ~/so2
-     $ git clone git@github.com:linux-kernel-labs/linux.git ~/so2/linux
```

Comment:
This seems like a reasonable change to me,  @dbaluta  @razvand what do you think?

Reply:
And, I forgot updating the previous line to `mkdir -p ~/src/`.  Will update soon.